interhost.py

```diff
@@ -304,6 +400,35 @@ def make_vcf(a, ref_idx, chrom):
                             vars.append(m+1)
             yield row+vars
             
+def transposeChromosomeFiles(inputFilesList, outputDirectory, outputFilePrefix, inputFormat="fasta", outputFormat="fasta"):
+    outputFilenames = []
+
+    # get BioPython iterators for each of the FASTA files specified in the input
+    fastaFiles = [SeqIO.parse(open(x, "rU"), inputFormat) for x in inputFilesList]
```

Comment:
Don't forget to close all these files before the end of transposeChromosomeFiles.  I think you'll have to do something like:

```
inFastas = list(open(x, 'rU') for x in inputFilesList)
fastaFiles = [SeqIO.parse(x, inputFormat) for x in inFastas]
```

... then at the end ...

```
for x in inFastas:
  x.close()
```